bluemira/geometry/tools.py

```diff
@@ -1257,14 +1256,10 @@ def deserialize_shape(buffer: dict):
     supported_types = [BluemiraWire, BluemiraFace, BluemiraShell]
 
     def _extract_mesh_options(shape_dict: dict):
-        mesh_options = None
+        mesh_options = meshing.MeshOptions()
```

Comment:
This won't return None if neither lcar or physical group is not in the shape_dict. Perhaps we don't want this anyway, but as is the refactor is not 1:1.

Reply:
sure I'll rework so its the same for now